packages/manager/modules/bm-server-components/src/order-private-bandwidth/private-order.controller.js

```diff
@@ -108,6 +107,15 @@ export default class BmServerComponentsOrderPrivateBandwidthCtrl {
         .then((result) => {
           this.model.orderUrl = result.order.url;
         })
+        .catch((error) => {
+          this.handleError(
+            error,
+            this.$translate.instant('server_error_bandwidth_order', {
+              error: error.data?.message,
+            }),
+          );
+          this.goBack(true);
+        })
```

Comment:
To avoid duplicating code, you can put that part in a dedicated function

```suggestion
        .catch(this.onError)
```

```
onError(error) {
 this.handleError(
            error,
            this.$translate.instant('server_error_bandwidth_order', {
              error: error.data?.message,
            }),
          );
          this.goBack(true);
}
```